### PR TITLE
Task app service deploy streaming log

### PR DESCRIPTION
**Pull request by ivywei0125**
What does this implement/fix? Explain your changes.
---------------------------------------------------
web-app-deploy-succeed-start-timeout

Does this close any currently open issues?
------------------------------------------

Any relevant logs, screenshots, error output, etc.?
-------------------------------------

Any other comments?
-------------------
…

Has this been tested?
---------------------------
- [ ] Tested

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-appservice-java/src/main/java/com/microsoft/azure/toolkit/intellij/legacy/function/runner/deploy/FunctionDeploymentState.java [hunk 1]
```diff
@@ -68,14 +74,24 @@ public FunctionDeploymentState(Project project, FunctionDeployConfiguration func
...
+        final IAzureMessager messenger = AzureMessager.getDefaultMessager();
+        OperationContext.current().setMessager(new RunProcessHandlerMessenger(processHandler));
         applyResourceConnection();
         final FunctionAppBase<?, ?, ?> target = FunctionAppService.getInstance().createOrUpdateFunctionApp(deployModel.getFunctionAppConfig());
         stagingFolder = FunctionUtils.getTempStagingFolder();
-        prepareStagingFolder(stagingFolder, processHandler, operation);
+        prepareStagingFolder(stagingFolder, operation);
         // deploy function to Azure
         FunctionAppService.getInstance().deployFunctionApp(target, stagingFolder);
+        AzureTaskManager.getInstance().runInBackground("list HTTPTrigger url", () -> {
```
Comment (wangmingliang-ms) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-appservice-java/src/main/java/com/microsoft/azure/toolkit/intellij/legacy/function/runner/deploy/FunctionDeploymentState.java:85]: use` OperationBundle.title`

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-appservice-java/src/main/java/com/microsoft/azure/toolkit/intellij/legacy/webapp/runner/webappconfig/WebAppRunState.java [hunk 2]
```diff
@@ -85,6 +91,7 @@ public WebAppRunState(Project project, WebAppConfiguration webAppConfiguration)
     @Override
     @AzureOperation(name = "user/webapp.deploy_artifact.app", params = {"this.webAppConfiguration.getWebAppName()"})
     public WebAppBase<?, ?, ?> executeSteps(@NotNull RunProcessHandler processHandler, @NotNull Operation operation) throws Exception {
+        final IAzureMessager messager = AzureMessager.getMessager();
```
Comment (wangmingliang-ms) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-appservice-java/src/main/java/com/microsoft/azure/toolkit/intellij/legacy/webapp/runner/webappconfig/WebAppRunState.java:94]: remove

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-springcloud/src/main/java/com/microsoft/azure/toolkit/intellij/springcloud/deplolyment/SpringCloudDeploymentConfigurationState.java [hunk 3]
```diff
@@ -140,6 +140,51 @@ private void printPublicUrl(final SpringCloudApp app) {
...
+            if (Objects.isNull(remote1)) {
+                return -1;
+            } else if (Objects.isNull(remote2)) {
+                return 1;
+            }
+            return StringUtils.compare(remote1.startTime(), remote2.startTime());
+        }).orElse(null);
+        if (Objects.isNull(appInstance)) {
+            return null;
+        }
```
Comment (wangmingliang-ms) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-springcloud/src/main/java/com/microsoft/azure/toolkit/intellij/springcloud/deplolyment/SpringCloudDeploymentConfigurationState.java:185]: can be reused with that in maven plugin.

**Review by wangmingliang-ms — DISMISSED** [review]
remember to resolve comments in later PRs.

**Review by Flanker32 — APPROVED** [review]
LGTM

**Pull request merged**